examples/vector_databases/Using_vector_databases_for_embeddings_search.ipynb

```diff
@@ -2409,13 +2460,250 @@
+   "source": [
+    "## Setup"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "96baabb9",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import openai\n",
+    "import json  \n",
+    "import openai\n",
+    "import wget\n",
+    "import pandas as pd\n",
+    "import zipfile\n",
+    "from azure.core.credentials import AzureKeyCredential  \n",
+    "from azure.search.documents import SearchClient  \n",
+    "from azure.search.documents.indexes import SearchIndexClient  \n",
+    "from azure.search.documents.models import Vector \n",
+    "from azure.search.documents.indexes.models import (  \n",
+    "    SearchIndex,  \n",
+    "    SearchField,  \n",
+    "    SearchFieldDataType,  \n",
+    "    SimpleField,  \n",
+    "    SearchableField,  \n",
+    "    SearchIndex,  \n",
+    "    SearchField,  \n",
+    "    VectorSearch,  \n",
+    "    VectorSearchAlgorithmConfiguration,  \n",
+    ") \n",
+    "\n",
+    "\n",
+    "openai.api_type = \"azure\"\n",
+    "openai.api_base = \"YOUR_AZURE_OPENAI_ENDPOINT\"\n",
+    "openai.api_version = \"2023-05-15\"\n",
+    "openai.api_key = \"YOUR_AZURE_OPENAI_KEY\"\n",
+    "model: str = \"text-embedding-ada-002\"\n",
+    "\n",
+    "search_service_endpoint: str = \"YOUR_AZURE_SEARCH_ENDPOINT\"\n",
+    "search_service_api_key: str = \"YOUR_AZURE_SEARCH_ADMIN_KEY\"\n",
+    "index_name: str = \"YOUR_AZURE_SEARCH_INDEX_NAME\"\n",
+    "credential = AzureKeyCredential(search_service_api_key)\n"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "0410b0a4",
+   "metadata": {},
+   "source": [
+    "## Create a search index"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b62cc101",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Create a search index\n",
+    "index_client = SearchIndexClient(\n",
+    "    endpoint=search_service_endpoint, credential=credential)\n",
+    "fields = [\n",
+    "    SimpleField(name=\"id\", type=SearchFieldDataType.String),\n",
+    "    SimpleField(name=\"vector_id\", type=SearchFieldDataType.String, key=True),\n",
+    "    SimpleField(name=\"url\", type=SearchFieldDataType.String),\n",
+    "    SearchableField(name=\"title\", type=SearchFieldDataType.String),\n",
+    "    SearchableField(name=\"text\", type=SearchFieldDataType.String),\n",
+    "    SearchField(name=\"title_vector\", type=SearchFieldDataType.Collection(SearchFieldDataType.Single),\n",
+    "                searchable=True, dimensions=1536, vector_search_configuration=\"my-vector-config\"),\n",
+    "    SearchField(name=\"content_vector\", type=SearchFieldDataType.Collection(SearchFieldDataType.Single),\n",
+    "                searchable=True, dimensions=1536, vector_search_configuration=\"my-vector-config\"),\n",
+    "]\n",
+    "\n",
+    "vector_search = VectorSearch(\n",
+    "    algorithm_configurations=[\n",
+    "        VectorSearchAlgorithmConfiguration(\n",
```

Comment:
Is this still right in the latest build of the SDK? In the C# version we did per-algo kind types, don't remember if we did the same in Python. Same with the hnsw_parameters property below.